examples/developer-tools/src/blocks/inputs.ts

```diff
@@ -0,0 +1,149 @@
+import * as Blockly from 'blockly';
+
+const fieldMessage = 'fields %1 %2';
+const fieldArgs = [
+  {
+    'type': 'field_dropdown',
+    'name': 'ALIGN',
+    'options': [['left', 'LEFT'], ['right', 'RIGHT'], ['centre', 'CENTRE']],
+  },
+  {
+    'type': 'input_statement',
+    'name': 'FIELDS',
+    'check': 'Field',
+  },
+];
+
+const typeMessage = 'type %1';
+const typeArgs = [
+  {
+    'type': 'input_value',
+    'name': 'TYPE',
+    'check': 'Type',
+    'align': 'RIGHT',
+  },
+];
+
+/**
+ * Check to see if more than one input has this name.
```

Comment:
Can you document the side effect that this method has? I.E. applying the warning.